src/utils/buildThemeForm.spec.ts

```diff
@@ -0,0 +1,12 @@
+import FormData from 'form-data';
+
+import { buildMultipartBody } from './buildMultipartBody';
+
+describe('buildImageForm', () => {
```

Comment:
Hey NeKisha, good job on the `buildMultipartBody` utility function. However, I noticed we still have two separate unit testing files, still named by the old function names. This is unnecessary, and a bit redundant. Unit test files are most typically associated with implementation files on a 1-to-1 basis (with some less common exceptions). We should get rid of both `buildThemeForm.spec.ts` as well as `buildImageForm.spec.ts` and instead replace them both with a file called `buildMultipartBody.spec.ts` that tests the following:
1. Passing in a fixture and a key name, we expect that `buildMultipartBody` will return a FormData object.
2. I would also recommend adding another test to test that the `key` parameter you recently added is properly added to the FormData object that is returned by the function.

Thanks!